MdeModulePkg/Bus/Pci/XhciDxe/Xhci.c

```diff
@@ -837,7 +837,13 @@ XhcTransfer (
   }
 
   Xhc->PciIo->Flush (Xhc->PciIo);
+  // MU_CHANGE [BEGIN] - Fix USB reset issue
+  //
+  // Do not free URB data, since it is passed in as an external argument
+  // and allocated and managed by the caller.
+  //
   XhcFreeUrb (Xhc, Urb);
+  // MU_CHANGE [END] - Fix USB reset issue

```

Comment:
nit: Could move this end to the end of the comment, so just future changes don't make this cherry-pick fail